std/zlib.d

```diff
@@ -315,7 +315,14 @@ class Compress
   public:
 
     /**
-     * Construct. level is the same as for D.zlib.compress(). header can be used to make a gzip compatible stream.
+     * Construct.
+     *
+     * Params:
+     *    level = the same as for $(D std.zlib.compress())
```

Comment:
The description also could be better. Why not copy the description of `level` from `compress()`? "the same as for X" is not a very helpful description.

Also, `$(LREF...)` should be used here so that the user can click on the link to see what exactly `std.zlib.compress` is, instead of having to navigate there manually.